src/platform/src/Result/TokenUsage/TokenUsage.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\AI\Platform\Result\TokenUsage;
+
+/**
+ * @author Junaid Farooq <ulislam.junaid125@gmail.com>
+ */
+final class TokenUsage implements \JsonSerializable
+{
+    public function __construct(
+        public ?int $promptTokens = null,
```

Comment:
```suggestion
        public ?int $prompt = null,
```
Maybe remove all the Token suffix here? They feel superfluous

Reply:
No problem.
I would not have kept the suffix if the object were simply named something like `Tokens`, but thought `TokenUsage` is a better name.

Reply:
To be discussed with @chr-hertel

Reply:
I did remove the suffixes BTW.

Reply:
`TokenUsage` for the class name and _tokens_ as the suffixes for the property names sounds more logical to me.
Or conversely, `Tokens` for the class name and no suffixes for the property names would also be a good option.

Reply:
TokenUsage and tokens as suffix sound best to me

Reply:
:man_facepalming: :smile: